website/docs/components/separator/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,30 @@
+
+## Placement
+
+Use separators to distinguish between sections on a page visually. Use them sparingly and purposefully to avoid cluttering the UI and distracting users from the main content.
+
+!!! Do
+
+Use to create a clear break between paragraphs, blocks, or sections.
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-do.png)
+
+!!!
+
+!!! Dont
+
+![Example of a leading icon in the tag](/assets/components/separator/separator-dont.png)
+
+!!!
+
+## Spacing
+
+We recommend using meaningful spacing between content elements, allowing users to scan and understand information quickly.
+
+### 0px spacing
+   - Use the `0px` spacing option for a tightly integrated layout without any visible gaps between elements.
+   - Avoid using the `0px` spacing option when the content sections need clear visual separation or when visual clutter is possible.
```

Comment:
Personally, for better readability, I would remove the "Avoid" paragraph here and in the section below (since they're saying the same thing of the "suggested" ones, only for the other variant).
Essentially I would merge the
- "use 0px" under the `0px` with "avoid 0px" under the `24px`
- "use 24px" under the `24px` with "avoid 24px" under the `0px`

Reply:
I see what @didoo is saying here. It's almost as if there's no need to have the two separate sections. Could you just have one bulleted list under the spacing heading with all the details? Something like: 

> We recommend using meaningful spacing between content elements, allowing users to scan and understand information quickly.
> 
> - Use the `0px` spacing option for a tightly integrated layout without any visible gaps between elements.
> - Use the `24px` spacing option to visually distinguish between different sections, making it simpler for users to navigate and understand the content.

Reply:
I do like this approach, Its cleaner. I'll update it now! Thank you @didoo and @heatherlarsen